addons/default/visiosoft/advs-module/resources/js/editContactInformation.js

```diff
@@ -1,30 +1,48 @@
 phoneMask("input[name='gsm_phone'],input[name='office_phone'],input[name='land_phone']")
 $('.formEditInfo').on('submit', function (e) {
     e.preventDefault();
+
+    const oldBtnHtml = $(this).find('.btn-success').html();
+    $(this).find('.btn-success').attr('disabled', true).html(`
+        <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+    `);
+
     var form = $(this);
 
-    crud(form.serialize() + "&action=update", '/ajax/update-user-info', 'POST', function (callback) {
```

Comment:
@spektra2147 lets not use crud anymore, I think just using $.ajax is much cleaner + there is no reason to use crud over $.ajax they are basically doing the same thing.